stages/org.osbuild.skopeo

```diff
@@ -20,7 +20,8 @@ def main(inputs, output, options):
             if dest_type == "containers-storage":
                 storage_root = destination.get("storage-path", "/var/lib/containers/storage")
                 storage_driver = destination.get("storage-driver", "overlay")
-                dest = f"containers-storage:[{storage_driver}@{output}{storage_root}+/run/containers/storage]{image_name}"
+                dest = f"containers-storage:[{storage_driver}@{output}{storage_root} \
+                    +/run/containers/storage]{image_name}"
```

Comment:
This introduced a bunch of spaces :) Probably better to just tell the linters that the long line is ok.

Reply:
Thanks for the review:) So if this would mean we shall change linter config for the whole project to allow long lines?

Reply:
I think checking if there is a way with autopep8 to disable a specific lines or blocks would be preferable. What is the issue exactly, is it a big too big for our limit (120iirc)?

Reply:
Also the content of this line is not tested afaict so any change like this is dangerous - also the `+` will be put verbatim into the code if just applied as is.

Reply:
I have removed this from PR to address line length issue in separated PR.